testing/trino-testing/src/main/java/io/trino/testing/AbstractTestEngineOnlyQueries.java

```diff
@@ -6728,15 +6732,15 @@ FUNCTION a(x integer) RETURNS integer RETURN b(x),
                   FUNCTION b(x integer) RETURNS integer RETURN x * 2
                 SELECT a(10)
                 """))
-                .failure().hasMessage("line 3:8: Function 'b' not registered");
+                .failure().hasMessage("line 2:48: Function 'b' not registered");
 
         // inline function cannot be recursive
         // note: mutual recursion is not supported either, but it is not tested due to the forward declaration limitation above
         assertThat(query("""
                 WITH FUNCTION a(x integer) RETURNS integer RETURN a(x)
                 SELECT a(10)
                 """))
-                .failure().hasMessage("line 3:8: Recursive language functions are not supported: a(integer):integer");
+                .failure().hasMessage("Recursive language functions are not supported: a(integer):integer");
```

Comment:
This one could have location marker, because it's an inline funciton.

Reply:
It could, but I'm not sure how to do that. We don't have access to the original AST at this point.